apis/r/DESCRIPTION

```diff
@@ -33,6 +33,9 @@ Imports:
     Matrix,
     stats,
     bit64,
+    # We can't pin on 0.17.*, 0.18.*, etc since CRAN forbids it.  This means that whenever tiledb-r
+    # bumps, we can have CI fails in the time window between that release and our bumping
+    # libtilesoma's core-dependency version.
```

Comment:
1.  We are not on CRAN.  This is not a current binding constraint, and may not for some time.  So why make it an issue.

2.  One cannot leave comments in the DESCRIPTION file.  Try `R CMD check .` on _any_ package where you add something like that.  It will blow up immediately.

I suggest closing and withdrawing the PR.